### PR TITLE
[AT-4967] Drop all live tests

### PR DESCRIPTION
**Pull request by seedlit**
Scope:
 - Dropped all the live tests and fixtures

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [x] Bumped version
* [x] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -133,15 +120,3 @@ workflows:
...
-    triggers:
-      - schedule:
-          cron: "0 0 * * *"
-          filters:
-            branches:
-              only:
-                - master
-    jobs:
-      - test_live:
-          context: up42-py-live-testing
```
Comment (javidq) [inline .circleci/config.yml:147]: Can we also delete this context?

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,12 @@ You can check your current version with the following command:
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
+## 0.38.0a1
```
Comment (javidq) [inline CHANGELOG.md:33]: 0.37.1a1

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 (insert scope and description here)
 
 Items:
-* [ ] Ran test & live-tests
 * [ ] Implemented (new) unit tests
```
Comment (javidq) [inline docs/PULL_REQUEST_TEMPLATE.md:4]: Next 3 can be dropped as well

File: tests/test_tasking.py [hunk 4]
```diff
@@ -7,7 +7,6 @@
 
 from .fixtures import fixtures_globals as constants
 
-LIVE_TEST_WORKSPACE_ID = os.getenv("LIVE_TEST_WORKSPACE_ID")
 LIVE_FEASIBILITY_ID = os.getenv("LIVE_FEASIBILITY_ID")
 LIVE_OPTION_ID = os.getenv("LIVE_OPTION_ID")
```
Comment (javidq) [inline tests/test_tasking.py:11]: These two need to drop LIVE prefix and change to str(uuid.uuid4())

File: .circleci/config.yml (hunk 1 above)
Comment (javidq) [reply .circleci/config.yml:147]: Or have a jira task to delete it

**Review by javidq — CHANGES_REQUESTED** [review]

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (seedlit) [reply docs/PULL_REQUEST_TEMPLATE.md:4]: Let's retain `Implemented (new) unit tests`?
Or is there a reason we want to remove it?

File: .circleci/config.yml (hunk 1 above)
Comment (seedlit) [reply .circleci/config.yml:147]: good catch!
deleted from circleCI

Comment (seedlit) [reply .circleci/config.yml:147]: actually I should have waited for this PR to be merged to master before deleting.
But should be fine though

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (javidq) [reply docs/PULL_REQUEST_TEMPLATE.md:4]: This is something already being checked by Sonar coverage

**Review by javidq — APPROVED** [review]

File: tests/test_catalog.py [hunk 5]
```diff
@@ -205,50 +151,6 @@ def test_search_usagetype(catalog_usagetype_mock):
...
-    e.g. when searching for ["DATA", "ANALYTICS"],
-    the result can be ["DATA"],
-    ["ANALYTICS"] or ["DATA", "ANALYTICS"].
-    """
-    search_parameters = catalog_live.construct_search_parameters(
-        start_date="2014-01-01T00:00:00",
-        end_date="2020-12-31T23:59:59",
-        collections=["phr"],
-        limit=100,
-        usage_type=usage_type,
```
Comment (janchrizz) [inline tests/test_catalog.py:233]: i wasn't aware of this `usage_type` param, didn't find it in the code reference docs. But anyways, will add it to my next PR

**Review by janchrizz — APPROVED** [review]
lgtm, tests are covered

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (seedlit) [reply docs/PULL_REQUEST_TEMPLATE.md:4]: but we didn't set it to 100% yet.
Let's retain it in the meantime.

**Pull request merged**